registry/storage/driver/s3-aws/s3.go

```diff
@@ -148,6 +149,11 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		return nil, fmt.Errorf("Invalid region provided: %v", region)
 	}
 
+	regionEndpoint, ok := parameters["regionendpoint"]
```

Comment:
i want both region and region endpoint to be provided because  it is required during creation of s3 client object.